### ISSUE TITLE
[QUESTION]unitless warning in HEMCO.log

### ISSUE THREAD

**Issue by XR-peking** [geoschem/geos-chem#1242]
### Ask a question about GEOS-Chem:
Why we should set all the unit of each parameter in meteorological files ( MERRA2.nc ) to unitless in HEMCO_Config.rc ?
![1652194051(1)](https://user-images.githubusercontent.com/103945599/167656830-144e1d1c-aaea-4b4d-9e89-4f17a179585a.png)

**Comment by msulprizio** [comment]
The units in HEMCO_Config.rc will be retired at some point. They were originally added so that HEMCO could apply automatic unit conversions behind-the-scenes. However, we found this to be a source of confusion so we have since removed automatic unit conversions and instead recommend explicitly applying scale factors in HEMCO_Config.rc to convert units. Leaving fields as unitless was one way to avoid having HEMCO attempt to convert units and instead pass fields as-is to GEOS-Chem.
